torch/fx/experimental/symbolic_shapes.py

```diff
@@ -281,6 +282,8 @@ def eval(cls, base, divisor):
 magic_methods_on_builtins = {"min", "max"}
 magic_methods_on_math = {"ceil", "floor"}
 magic_methods_on_submodule = {"sym_float", "sym_sqrt"}
+# operator.__and__ exists, operator.and does not.
+operator_dunder_only = {"and"}
```

Comment:
is `__or__` also like this? Might be good to do a quick scan through the operator docs.